tools/cell_census_builder/tests/test_util.py

```diff
@@ -0,0 +1,106 @@
+import numpy as np
+import pytest
+from scipy.sparse import coo_matrix, csr_matrix, triu
+
+from tools.cell_census_builder.util import array_chunker, is_positive_integral, uricat
+
+
+def test_is_positive_integral() -> None:
+    X = np.array([1, 2, 3, 4])
```

Comment:
these result in an array of dtype np.int64. The function only accepts floats (both float32 and float64), although it does not raise an error if the input violates the type signature.

These tests should explicitly create arrays of dtype float32 or float64.  Prefer float32 for most tests, as that is what we typically receive.